guides/micronaut-oauth2-auth0/java/src/main/java/example/micronaut/HomeController.java

```diff
@@ -15,7 +15,13 @@ public class HomeController {
     @Secured(SecurityRule.IS_ANONYMOUS) // <2>
     @View("home") // <3>
     @Get // <4>
-    public Map<String, Object> index() {
-        return new HashMap<>();
+    public HttpResponse<Map<String, Object>> index(@Nullable Authentication authentication) {
```

Comment:
This change should not be necessary. 

We have a View Model Processor which includes the security in the model map. 

https://github.com/micronaut-projects/micronaut-views/blob/master/views-core/src/main/java/io/micronaut/views/model/security/SecurityViewModelProcessor.java

I am surprised it did not work for you. 

I will be happy to show more information in the home.html screen but I would like not do changes to the controller if possible.

Reply:
Ok, now, I know why the parameter wasn't reachable in the `Thymeleaf` template. I was returning en empty response `return HttpResponse.ok();`. In that case, `${security.parameters}` is empty.
Nevertheless, I founded useful to know that we can access information in the controller and in the template. But if you don't, I understand. 

Thank you for the link to `SecurityViewModelProcessor`